src/tbbmalloc_proxy/proxy.cpp

```diff
@@ -59,6 +59,13 @@
 // Use MallocMutex implementation
 typedef MallocMutex ProxyMutex;
 
+// Adds aliasing and copy attributes to function if available
+#if __has_attribute(__copy__)
```

Comment:
Not all compilers recognize `__has_attribute`, i will wrap it into `#if defined __has_attribute/#endif` (as recommend in https://gcc.gnu.org/onlinedocs/cpp/_005f_005fhas_005fattribute.html)

Reply:
Yes that's a v good idea, thank you!